Better C# Code

Harcourt1 opened the pull request:
Fixed bad C# code.

Harcourt1 committed:
Better C# Code

Fixed bad C# code.

Crow-Control reviewed (changes requested):
Looks decent, although those spaces needs to be turned into tabs before merge.

Harcourt1 committed:
Changed to tabs.

I suggest, in the final version make sure all code is in single space indents. Its saves space.

Harcourt1 committed:
yea yea yea tabs

Im not a huge fan of this new version. VIM hates me. 
In the final version everything should be 1 space indents due to it saving space
Plz Merge Now
